experimental/js-module-blocks.md

```diff
@@ -0,0 +1,14 @@
+# [JS Module Blocks][proposal-js-module-blocks]
+
+## Expressions
+
+### ModuleExpression
+
+```js
+interface ModuleExpression <: Expression {
+    type: "ModuleExpression";
+    body: Program;
+}
+```
```

Comment:
Consider add a note that the `sourceType` of `body` must be `"module"`.

Reply:
We can probably subtype `Program` to a `Module` for that. But it's fine for experimental.